app/routes/user.email._index/EmailNotificationCard.tsx

```diff
@@ -151,6 +154,51 @@ export default function EmailNotificationCard({
           </ModalToggleButton>
         </ModalFooter>
       </Modal>
+      <Modal
+        id="test-confirmation"
+        ref={testConfirmRef}
+        aria-labelledby="modal-confirm-heading"
+        aria-describedby="modal-confirm-description"
+        renderToPortal={false} // FIXME: https://github.com/trussworks/react-uswds/pull/1890#issuecomment-1023730448
+      >
+        <ModalHeading id="modal-confirm-heading">
+          Test Email Confirmation
+        </ModalHeading>
+        <p id="modal-confirm-description">
+          This will send a generic test email to {recipient}. Do you wish to
+          continue?
+        </p>
+        <ReCAPTCHA
+          onChange={(value) => {
+            setRecaptchaValid(Boolean(value))
+          }}
+        />
```

Comment:
The ReCAPTCHA component probably has to be inside the form.

Reply:
Woops, fixed in latest commit